src/main/java/net/education/kyivstar/benchmark/Logic.java

```diff
@@ -0,0 +1,6 @@
+package net.education.kyivstar.benchmark;
+
+public interface Logic {
```

Comment:
Change to Task for better clarity. This name is simple and indicates that it represents a unit of work.
Also, rename all implementing Class